src/pymmcore_widgets/useq_widgets/_z.py

```diff
@@ -315,11 +312,11 @@ def setMode(
         self._on_change()
 
     def mode(self) -> Mode:
-        """Return the current mode."""
+        """Return the current mode, one of "top_bottom", "range_around", or "above_below"."""  # noqa: E501
```

Comment:
i don't want us to get in the habbit of using `# noqa: E501` whenever we want have a long line.  E501 is a good one, and it's there to enforce this.  In this, if you want to give more info, put it on another line